changelog/dev-update-workflows

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Update GH workflows to use PHP version from plugin file.
```

Comment:
It should be a non-significant changelog, so it doesn't appear on the next release's changelog